examples/html-kitchen-sink/tests/__snapshots__/htmlshots.test.js.snap

```diff
@@ -61,67 +61,6 @@ exports[`Storyshots Addons|Actions Multiple actions, selector 1`] = `
         
 `;
 
-exports[`Storyshots Addons|Actions story1 1`] = `
-<button
-  type="button"
->
-  Hello World
-</button>
-`;
-
-exports[`Storyshots Addons|Actions story2 1`] = `
-<button
-  type="button"
->
-  Hello World
-</button>
-`;
-
-exports[`Storyshots Addons|Actions story3 1`] = `
-<button
-  type="button"
->
-  Hello World
-</button>
-`;
-
-exports[`Storyshots Addons|Actions story4 1`] = `
-<button
-  type="button"
->
-  Hello World
-</button>
-`;
-
-exports[`Storyshots Addons|Actions story5 1`] = `
-
-        
-`;
-
-exports[`Storyshots Addons|Actions story6 1`] = `
-<button
-  type="button"
->
-  Hello World
-</button>
-`;
-
-exports[`Storyshots Addons|Actions story7 1`] = `
-<button
-  type="button"
->
-  Hello World
-</button>
-`;
-
-exports[`Storyshots Addons|Actions story8 1`] = `
-<button
-  type="button"
->
-  Hello World
-</button>
-`;
-
```

Comment:
But we still have those stories: https://github.com/storybookjs/storybook/blob/2f6fad1e6dc240cf0907fc92932939734a8b4bc8/examples/html-kitchen-sink/stories/addon-actions.stories.js

Why did you accept that diff?

Reply:
We do?

I ran the tests after deleting them on `next` and it didn't add them back, so I figured they were old lingering snapshots

Reply:
Why would they be deleted?

Reply:
> Why would they be deleted?

No idea, maybe it has something to do with the new module format

Reply:
OK looks like those were duplicate snapshots stored under exported function name instead of story name